src/Games/NexusMods.Games.StardewValley/Resources/DiagnosticsTests/DisabledRequiredDependency.txt

```diff
@@ -0,0 +1,4 @@
+1. Install SMAPI
+2. Install v1.9.2 of Content Patcher: https://www.nexusmods.com/Core/Libs/Common/Widgets/DownloadPopUp?id=18130&game_id=1303&nmm=1
+3. Install Farm Type Manager: https://www.nexusmods.com/Core/Libs/Common/Widgets/DownloadPopUp?id=117244&game_id=1303
+4. Disable Content Patcher.
```

Comment:
So these are just repro steps for triggering a specific diagnostic?

Reply:
Yep, there's not really a better way as a lot of it relies on SMAPI server data that is hard to fake.